packages/components/src/tools-panel/test/index.tsx

```diff
@@ -1115,9 +1148,10 @@ describe( 'ToolsPanel', () => {
 			await openDropdownMenu();
 
 			const resetAllItem = await screen.findByRole( 'menuitem', {
-				disabled: false,
+				name: 'Reset all',
 			} );
 			expect( resetAllItem ).toBeInTheDocument();
+			expect( resetAllItem ).toHaveAttribute( 'aria-disabled', 'false' );
```

Comment:
Any reason to not use `.not.toBeDisabled()` here?

Reply:
Same as https://github.com/WordPress/gutenberg/pull/48275#discussion_r1113386000